vhdbuilder/packer/install-dependencies.sh

```diff
@@ -195,6 +195,32 @@ EOF
 
 echo "${CONTAINER_RUNTIME} images pre-pulled:" >> ${VHD_LOGS_FILEPATH}
 
+installAndConfigureArtifactStreaming() {
```

Comment:
do you need a seperate function for mariner?

Reply:
Depends on how the mirror will handle it. Currently it's installing for 22.04 only: https://github.com/juliusl/lifec_registry/blob/7ae610b456d48d17788917a23fa340ee54d60f57/lib/sh/overlaybd/install.sh#L6, so we will need to change that first. 

Mariner VHD builds are also done in this script, so we can use the same function for both Ubuntu and Mariner.